cli-files/go-ls/cmd/root_test.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestGolsCmdFunc(t *testing.T) {
+	t.Run("No path provided", func(t *testing.T) {
+		r, w, _ := os.Pipe()
+		os.Stdout = w
+
+		cmd := golsCmd
+		cmd.Execute()
```

Comment:
If an error happens, what will happen to the test?

Reply:
Got it, updated to add t.Fatal